content/docs/intro/languages/dotnet.md

```diff
@@ -0,0 +1,94 @@
+---
+title: ".NET"
+menu:
+  intro:
+    parent: languages
+    weight: 2
+
+aliases: ["/docs/reference/dotnet/"]
+---
+
+{{< langchoose dotnetonly >}}
+
+Pulumi supports programs written in any supported .NET language, including C#, F#, and Visual Basic. You can also use your favorite .NET tools such as editors, package managers, build systems, and test frameworks.
+
+## Getting Started
+
+### Prerequisites
+
+1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+1. [Install .NET Core 3.0+](https://dotnet.microsoft.com/download)
+
+### Example .NET Project
+
+You can get started with .NET using a Pulumi template. From an empty directory, create a new .NET project:
+
+```bash
+$ mkdir myproject && cd myproject
+$ pulumi new csharp
+```
+
+The template will autogenerate a set of files that make up a Pulumi project.
+
+{{% lang csharp %}}
+
+## C\#
+
+You can write Pulumi programs in C# to get additional verification and tooling benefits. As of version 1.5, Pulumi supports .NET natively. The fastest way to get started with Pulumi in C#, is to use a template:
+
+```bash
+$ mkdir myproject && cd myproject
+$ pulumi new csharp
+```
```

Comment:
This just repeats what was above.  Not clear we need this extra level of detail.  If we remove the lang chooser, not clear we need any section at all for C#, as it's just the "normal" case.  We can then include an F# specific section with details on what you need to do to use F#.

Reply:
accepted